themes/gohugoioTheme/layouts/index.html

```diff
@@ -14,6 +14,9 @@
   <section class="w-100 ph4 ph5-ns pv4 pv6-ns mid-gray bg-white bb bt b--light-gray">
     {{- partial "home-page-sections/installation" . -}}
   </section>
+
+  {{ partial "home-page-sections/sponsors.html" (dict "cx" . ) }}
```

Comment:
Why pass a `dict` here?

Reply:
So I can reuse the same partial in the footer and pass through some classes